jupyterhub.d/jupyterhub_config/docker.py

```diff
@@ -49,7 +49,7 @@
 
 # Proxy
 # Wrap the start of the proxy to allow bigger headers in nodejs
-c.ConfigurableHTTPProxy.command = '/srv/jupyterhub/jh_gitlab/scripts/start_proxy.sh'
+c.ConfigurableHTTPProxy.command = ['configurable-http-proxy', '--redirect-port', '80', '--redirect-to', '443']
```

Comment:
Can you also change in the other configs?

I'm not sure we need this, since SWAN is behind a reverse proxy. In k8s we probably don't need as well, or do we?

Reply:
If we install SwanNotificationService, it needs to be added as well.

Reply:
docker.py is not used by SWAN K8s installation, probably required for sciencebox ?

Reply:
docker.py is for the docker compose version of the configuration.

But this configuration for the proxy should be consistent in the k8s config versions as well. But I'm not sure is even necessary because I think that 1st) we always have apache in front and 2nd) we also have a reverse proxy for sciencebox. In that case, we can try to remove it completely.